src/ert/storage/local_ensemble.py

```diff
@@ -442,36 +419,3 @@ def save_response(self, group: str, data: xr.Dataset, realization: int) -> None:
         Path.mkdir(output_path, parents=True, exist_ok=True)
 
         data.to_netcdf(output_path / f"{group}.nc", engine="scipy")
-
-    def save_field(
-        self,
-        parameter_name: str,
-        realization: int,
-        data: Union[
-            npt.NDArray[np.double], np.ma.MaskedArray[Any, np.dtype[np.double]]
-        ],
-    ) -> None:
-        output_path = self.mount_point / f"realization-{realization}"
-        Path.mkdir(output_path, exist_ok=True)
-        if not np.ma.isMaskedArray(data):  # type: ignore
-            grid_path = self.experiment.grid_path
-            if grid_path is None:
-                raise ConfigValidationError(
-                    f"Missing path to grid file for realization-{realization}"
-                )
-            mask, shape = read_mask(grid_path)
-            if mask is not None:
-                data_full = np.full_like(mask, np.nan, dtype=np.double)
-                np.place(data_full, np.logical_not(mask), data)
-                data = np.ma.MaskedArray(
-                    data_full, mask, fill_value=np.nan
-                )  # type: ignore
-            else:
-                data = np.ma.MaskedArray(data.reshape(shape))  # type: ignore
-
-        np.save(f"{output_path}/{parameter_name}", data.filled(np.nan))  # type: ignore
-        self.update_realization_state(
-            realization,
-            [RealizationStateEnum.STATE_UNDEFINED],
-            RealizationStateEnum.STATE_INITIALIZED,
-        )
```

Comment:
Where do we update this info now?

Reply:
The function `save_parameters` in `local_ensemble.py` does this:

```python
        self.update_realization_state(
            realization,
            [
                RealizationStateEnum.STATE_UNDEFINED,
                RealizationStateEnum.STATE_LOAD_FAILURE,
            ],
            RealizationStateEnum.STATE_INITIALIZED,
        )
```

This seems to give the same behavior.